### DISCUSSION TITLE
Move Spaces above Docs sites - or allow the sections to be moved

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#677] [Feature Requests]
For my day-to-day work, I rarely need to touch the doc sites - they're don't need to be set up or updated all that frequently. Anyone accessing the published sites does so through the URL.

I do always work in the spaces.

Having **Docs sites** above **Spaces** is very inconvenient. Every morning I have to remember to collapse the **Docs sites** section to get it out of my way.

Can **Docs sites** be moved below **Spaces**? Or can we be allowed to set the display order of those sections? Or automatically collapse one or the other?

**Comment by SamyPesse** [comment]
We'll soon ship an improvement where collapsing the docs sites section will be remembered between page reloads.

**Reply by barneska** [reply]
Will it also ever be possible to re-order the docs sites list?  There are about 5 sites that I manage and probably 40 or more that others in my org manage.  The order they are in appear totally random.  Alpha order, visual grouping, and/or the ability to re-organize them so I can lump mine together will save me from hunting to find where my sites are amongst all the other sites.

**Reply by JaniceManwiller** [reply]
That is a little weird, especially since you can drag and drop the doc spaces to re-order them. I would also like to reorder the sites, since the doc site at the top of the list is sort of standalone that isn't really connected to the other ones and would prefer it out of the way at the bottom of the list.
